example/provider-extensions/seed/create-seed.sh

```diff
@@ -35,12 +35,13 @@ garden_kubeconfig=$1
 seed_kubeconfig=$2
 seed_name=$3
 
-registry_domain=$(cat "$SCRIPT_DIR"/registrydomain)
+registry_domain_file="$SCRIPT_DIR/registrydomain"
 seed_values=values.yaml
 if [[ "$seed_name" != "provider-extensions" ]]; then
-  registry_domain=$(cat "$SCRIPT_DIR"/registrydomain-"$seed_name")
+  registry_domain_file="$SCRIPT_DIR/registrydomain-$seed_name"
   seed_values=values-"$seed_name".yaml
 fi
+registry_domain=$(cat $registry_domain_file)
```

Comment:
```suggestion
registry_domain=$(cat "$registry_domain_file")
```
For people who have spaces in their path.